src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6839,6 +6839,30 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_SimpleProgramIsEmpty" xml:space="preserve">
     <value>At least one top-level statement must be non-empty.</value>
   </data>
+  <data name="ERR_Line_does_not_start_with_the_same_whitespace_as_the_last_line_of_the_raw_string_literal" xml:space="preserve">
```

Comment:
Let's stick with existing naming conventions

Reply:
i genuinely can't tell what the exint naming convention is.  It looks to be a mix of just capitalizing words, but pickign a different set of words than the actual error message.  How do you determine the right loc name at the compiler layer?

Reply:
The convention is camel-case and we try to stay short. I'd propose:

ERR_WrongLinePrefix
ERR_DirectivesDisallowRawStrings
ERR_DelimiterDisallowedInsideRawString
ERR_IncorrectDelimiter
ERR_UnterminatedRawString
// IDS_FeatureRawStringLiterals
ERR_InterpolatedStringsDisallowRawStrings
ERR_EmptyRawString

Reply:
i don't like the short strings because there's no good way to know in the code waht message the user is going to get.  Hovering over the enum member just gives you the constant integral value.  You have to go manually find the resx file and look inside to see what it is.  It seems to just add more steps to the most critical thing i care about with an error.  Also, it's nearly impossible to understand if there are placeholders to insert values to tell the user.

Reply:
We try to keep identifiers short.
Usually if the identifier is clear enough, but if I have do a FindAllRefs on the error code, which brings up where it's produced and the tests that produce it.
Let's stick with existing name conventions